templates/_search-form.html

```diff
@@ -1,3 +1,3 @@
-<form action="/search-results">
```

Comment:
Removing this form that causes the redirect

Algolia does not depend on it https://community.algolia.com/places/examples.html#simple-input